test/02.basic-querying.test.js

```diff
@@ -4,7 +4,7 @@ var db, User, Customer, AccessToken, Post, PostWithId, Category, SubCategory;
 
 /*eslint no-console: "off"*/
 /*global getSchema should*/
-describe('basic-querying', function () {
+xdescribe('basic-querying', function () {
```

Comment:
@wolfgang-s I won't turn it off, it make sense to leave it active as it doesn't cause any problem because with both refresh as true and false these test cases would pass successfully. So before merging this I would revert it and make it as active.